examples/rfm9x_transmit.py

```diff
@@ -0,0 +1,59 @@
+# Example to send a packet periodically
+# Author: Jerry Needell
+#
+import time
+import board
+import busio
+import digitalio
+import adafruit_rfm9x
+
+transmit_interval=10
```

Comment:
Could you add a comment above L10 describing this interval along with the unit (seconds? milliseconds?)

Reply:
implemented